Blish HUD/GameServices/Input/Mouse/MouseHandler.cs

```diff
@@ -74,8 +76,8 @@ public class MouseHandler : IInputHandler {
             MouseState rawMouseState = Mouse.GetState();
 
             this.State = new MouseState(
-                                        (int)(rawMouseState.X / GameService.Graphics.UIScaleMultiplier),
-                                        (int)(rawMouseState.Y / GameService.Graphics.UIScaleMultiplier),
+                                        (int) (rawMouseState.X / GameService.Graphics.UIScaleMultiplier),
+                                        (int) (rawMouseState.Y / GameService.Graphics.UIScaleMultiplier),
```

Comment:
Any reason to continue to use the mouse position we're polling for if we've got the raw position?

Reply:
See here https://github.com/blish-hud/Blish-HUD/pull/290#discussion_r465554001